i18n/camellia_world.csv

```diff
@@ -1,74 +1,76 @@
+t0448179920,"Too bad, have a nice day.","残念ながら、良い一日を。"
+t4167f6fbca,No...,え…
+tb50e15cdf4,"hey, I didn't mean it like that! I meant I just forgot the ingredients... at home, yes, at home!","そんなつもりじゃなかった！ 材料を忘れただけってことだ…家に、そう、家に！"
+t27602b0a59,I can't leave in case customers show up! Can you get some new ingredients for me instead?, "代わりに材料を持って来てもらえる？お客さんが来るかもしれないから帰れないから！ "
+tb5eb062088,Sure...,わかった…
+td47c41e60d,That doesn't make any sense!,意味わかんない！
+t903573f4e2,"that's not a no, so...",断っていないので…
 td48582b880,"so the ingredients that are missing are:
 - Pizza dough
-- Pineapples",
-t1abf754fca,Pineapple on Pizza?!,
-tbf23a1e6be,it's delicious!,
-tdd5899cb7c,I mean if I get my pizza...,
-tb37eb9f88d,I'll even give you the pizza for free,
-t8e0a5fc21e,"So, I was pretty sure I had the [b]Pizza dough[/b] somewhere, but I can't seem to find it. It should still be somewhere around here.",
-teb5425954f,"For the [b]Pineapples[/b], some rude customer threw my Pineapples all the way over there onto these stone pillars. Since you got wings, can you maybe fly up there and grab them?",
-t3c9523baa9,Thank you so much for the ingredients!,
-t699c431a53,I'll make you a pizza right away...,
-taf58803057,"*bzzzzt* [wait=2.0]*[do SFX.play_wav(""res://sfx/pickup_coin.wav"")]DING*",
-t65869d2f2f,"Here you go! Pizza with extra pineapple, for free, as promised.",
-t4f01ba9414,"I've got pizza again, but nobody wants any... More for me!",
-t9322cd252e,The future of flavor is TASTE INFINITE,
-t7496164702,I've got a letter from Kassan for you,
-t0c3cbf846d,"Super dope, thank you!",
+- Pineapples","足りない材料は：
+- ピザ生地
+- パインアップル"
+t1abf754fca,Pineapple on Pizza?!,ピザにパインアップル？！
+tbf23a1e6be,it's delicious!,おいしいよ！
+tdd5899cb7c,I mean if I get my pizza...,ピザもらっても…
+tb37eb9f88d,I'll even give you the pizza for free,無料でピザあげるから
+t8e0a5fc21e,"So, I was pretty sure I had the [b]Pizza dough[/b] somewhere, but I can't seem to find it. It should still be somewhere around here.","たしか、[b]ピザ生地[/b]あったが、見つからない。ここらへんにあるはず。"
+teb5425954f,"For the [b]Pineapples[/b], some rude customer threw my Pineapples all the way over there onto these stone pillars. Since you got wings, can you maybe fly up there and grab them?","[b]パインアップル[/b]について…失礼な客さんがそこの石柱の上に投げた。翼あるから、上に飛んで、[b]パインアップル[/b]を拾ってくれる？"
+t3c9523baa9,Thank you so much for the ingredients!,材料本当にありがとな！
+t699c431a53,I'll make you a pizza right away...,すぐにピザを焼きます…
+taf58803057,"*bzzzzt* [wait=2.0]*[do SFX.play_wav(""res://sfx/pickup_coin.wav"")]DING*","*ビィィィィー* [wait=2.0]*[do SFX.play_wav(""res://sfx/pickup_coin.wav"")]チン*"
+t65869d2f2f,"Here you go! Pizza with extra pineapple, for free, as promised.",はいよ！約束通り、無料のパインアップルたっぷりのピザ。
+t4f01ba9414,"I've got pizza again, but nobody wants any... More for me!","またピザをあるけど、誰も私のピザを欲しがらない…もっと私の食べる分！"
+t9322cd252e,The future of flavor is TASTE INFINITE,"味の未来は「TASTE INFINITE」"
+t7496164702,I've got a letter from Kassan for you,かっさんからの手紙持っています
+t0c3cbf846d,"Super dope, thank you!","マジで？ありがとう！"
```

Comment:
I had this from the lyrics of the song, need to check what that came from and might update it later